core/lib/generators/spree/dummy/dummy_generator.rb

```diff
@@ -44,6 +44,7 @@ def generate_test_dummy
       opts[:skip_yarn] = true
       opts[:skip_bootsnap] = true
       opts[:skip_webpack_install] = true
+      opts[:skip_javascript] = true
```

Comment:
Is `skip_webpack_install` needed if we have `skip_javascript`?

Reply:
Actually not! 👍 
Pushing the `skip_webpack_install` removal.

Reply:
Reference https://github.com/rails/rails/blob/6-1-stable/railties/lib/rails/generators/app_base.rb#L408